knack/auto-zoom.js

```diff
@@ -0,0 +1,49 @@
+(function() {
+  var startingTime = new Date().getTime();
+  // Import jQuery into this file from CDN
+  // https://stackoverflow.com/questions/34338411/how-to-import-jquery-using-es6-syntax
+  var script = document.createElement("SCRIPT");
+  script.src =
+    "https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js";
+  script.type = "text/javascript";
+  document.getElementsByTagName("head")[0].appendChild(script);
+
+  // Create polling function for made sure jQuery is loaded and ready...
+  var checkReady = function(callback) {
```

Comment:
I'd like to talk through this function because I'm not familiar with polling, and I think it might be strategy that could help with a bug in the React side.

Reply:
yeah sounds good!